### ISSUE TITLE
Question about demo

### ISSUE THREAD

**Issue by srd-software** [VuioVuio/DelphiUCL#1]
First, your components and work on a form with a viable custom titlebar is _great_. Thanks.

Question about the demo: I'd like to make the caption panel change color depending on if the app is active or not. I was able to do it easily in my own test app, but I wasn't able to get it to do that in the demo app.

Also, at line 286 in the UCL.QuickButton file, I made this change:
```
qbsMax:
  if not FullScreen then
    begin
      if ParentForm.WindowState <> wsNormal then
      begin
        ParentForm.WindowState := wsNormal;
        Caption := #$E922; //normal win10 icon
      end
      else
      begin
        ParentForm.WindowState := wsMaximized;
        Caption := #$E923; //maximized win10 icon
       end;
     end;
```
I did that so the max/restore icons look like the normal ones in the OS:

![image](https://user-images.githubusercontent.com/51385541/76863898-daabeb00-6825-11ea-8f28-782c5a8cde76.png)

![image](https://user-images.githubusercontent.com/51385541/76863918-e0093580-6825-11ea-89b0-95dca7aaa5fb.png)

Is the change I made and where I put it what you would recommend to achieve this? 

Thanks

**Comment by srd-software** [comment]
I got the color change fiqured out. I needed to add CustomBackColor.Enabled := true; in my activate and deactivate code to get it to work.

**Comment by ghost** [comment]
I will consider changing the max/restore icon. There are still some minor problems with it.
Best regards.

**Comment by srd-software** [comment]
Thanks. Probably something like intercepting min/max messages will work. For now it's possible to implement it in the app it's being used in. Just be more convenient if the button could handle it.

I've been trying to figure out how to do a custom titlebar like this for a long time (since WinXP days). I wanted a form that behaved like a normal form and had a titlebar with colors I could set and add buttons, labels images or whatever to it, but I always fell back on the bsNone form style to get things working. The VCL styles now available in the IDE are OK, but super hard to modify (at least I find them hard to change). And I prefer to handle "theme" type stuff (especially colors) myself.

I've had some success with this kind of thing but never could get it really nailed down like you have. I was able to make a custom titlebar using DWM so the original OS min/max/close buttons continued to work along with images for any extra "buttons". I got that working well but I was never able to figure out how to do solid colors on it (plus it's a LOT of code to deal with). 

What you have come up with is the first really good solution I've found that is easy to implement and use. As far as colors go, I'm at the point where I just want to be able to set a "light" theme, a "dark" theme, or set the caption color to whatever the system setting is. I've done apps where I have all sorts of theme colors and while it looks good, it's so hard to maintain that I don't know if it's worth it anymore!

I've always gotten a lot of the same hits with web searches in the past when searching for custom titlebars or forms with no captions. I was going down this road again when I happened to come across your stuff a couple of days ago.

Thanks again for coming up with something that actually works!
